omisego-sdk/src/main/java/co/omisego/omisego/security/KeyManager.kt

```diff
@@ -0,0 +1,57 @@
+package co.omisego.omisego.security
+
+import android.content.Context
+import android.util.Base64
+import java.security.KeyStore
+import javax.crypto.Cipher
+
+/**
+ * OmiseGO
+ *
+ * Copyright © 2017 OmiseGO. All rights reserved.
```

Comment:
Would also suggest moving the licence as header of the file (before the package declaration) as well as making it a comment tag instead of a javadoc/kdoc tag 😄